gsoc/config.py

```diff
@@ -0,0 +1,14 @@
+import ruamel.yaml
+import os
+
+DATA_DIR = os.path.join(
+    os.path.dirname(__file__), '..',
+    '_site'
+)
+
+def load_cache(filename):
+    with open(os.path.join(DATA_DIR, filename), 'r') as f:
+        return ruamel.yaml.load(f, Loader=ruamel.yaml.Loader)
+
+def get_year():
+    return "2017"
```

Comment:
EOL at EOF.

but more importantly, returning 2017 here is inherently wrong.  the year is now 2018.  The GSoC 2018 program has now started.

And you want `2018 - 1`